README.md

```diff
@@ -34,15 +34,15 @@ The Rarimo SDK is a library that consists of many smaller NPM packages within th
-| [@rarimo/providers-evm](https://rarimo.github.io/js-sdk/modules/_rarimo_providers_evm.html)           | The EVM wallet wrappers called as Provider in the Rarimo SDK that can be used to interact with the EVM compatible blockchains                     | [![npm version](https://img.shields.io/npm/v/@rarimo/providers-evm.svg)](https://www.npmjs.com/package/@rarimo/providers-evm)           |
-| [@rarimo/providers-solana](https://rarimo.github.io/js-sdk/modules/_rarimo_providers_solana.html)     | The Solana wallet wrappers called as Provider in the Rarimo SDK that can be used to interact with the Solana blockchain                           | [![npm version](https://img.shields.io/npm/v/@rarimo/providers-solana.svg)](https://www.npmjs.com/package/@rarimo/providers-solana)     |
-| [@rarimo/providers-near](https://rarimo.github.io/js-sdk/modules/_rarimo_providers_near.html)         | The Near wallet wrappers called as Provider in the Rarimo SDK that can be used to interact with the Near blockchain                               | [![npm version](https://img.shields.io/npm/v/@rarimo/providers-near.svg)](https://www.npmjs.com/package/@rarimo/providers-near)         |
+| [@rarimo/shared](https://rarimo.github.io/js-sdk/modules/_rarimo_shared.html)                         | Utility functions, types, and constants shared across @rarimo packages.                                                                           | [![npm version](https://img.shields.io/npm/v/@rarimo/shared.svg)](https://www.npmjs.com/package/@rarimo/shared)                         |
+| [@rarimo/bridge](https://rarimo.github.io/js-sdk/modules/_rarimo_bridge.html)                         | Functionality for token bridging                                                                                                                  | [![npm version](https://img.shields.io/npm/v/@rarimo/bridge.svg)](https://www.npmjs.com/package/@rarimo/bridge)                         |
+| [@rarimo/swap](https://rarimo.github.io/js-sdk/modules/_rarimo_swap.html)                             | Tools for token swaps                                                                                                                             | [![npm version](https://img.shields.io/npm/v/@rarimo/swap.svg)](https://www.npmjs.com/package/@rarimo/swap)                             |
+| [@rarimo/provider](https://rarimo.github.io/js-sdk/modules/_rarimo_provider.html)                     | A common interface for wallets on different EVM and non-EVM chains                                                                                | [![npm version](https://img.shields.io/npm/v/@rarimo/provider.svg)](https://www.npmjs.com/package/@rarimo/provider)                     |
```

Comment:
descriptions here differ from the description in the related packages `package.json` files, is it ok? if not recheck plz, description in the package.json files will be shown on the npm site

Reply:
Made these consistent across package.json and readme files.